api.planx.uk/modules/send/email/index.ts

```diff
@@ -27,10 +27,11 @@ export async function sendToEmail(
   }
 
   try {
-    // Confirm this local authority (aka team) has an email configured in teams.submission_email
-    const { sendToEmail, notifyPersonalisation } =
+    // Confirm this local authority (aka team) has an email configured in team_settings.submission_email
+    const { notifyPersonalisation } =
```

Comment:
nit: If we're only using the `sendToEmail` field in this file, and not the rest of the `notifyPersonalisation` object, it would be cleaner / neater to destructure one level deeper here, and then not have to repeatedly write `notifyPersonalisation.sendToEmail` throughout the file 👍 

```suggestion
    const { notifyPersonalisation: { sendToEmail } } =
```

Reply:
I think we are also using `...notifyPersonalisation` for part of the code below. I'll keep as is now.